specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -86,154 +86,45 @@ _Jakarta™ Concurrency Specification, Version 1.0_. Available at: _https://jaka
 _Jakarta™ Batch Specification, Version 1.0_. Available at: _https://jakarta.ee/specifications/batch/1.0_
 
 Extension Mechanism Architecture, Available at
-_http://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_
-.
+_http://docs.oracle.com/javase/8/docs/technotes/guides/extensions/index.html_.
```

Comment:
minor nit but we should probably update these to https. I'm happy to create a separate PR for that if you agree.

Reply:
I'll just do it.  I'm already making other changes.